src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -192,6 +193,11 @@ function validate_options_type_by_value(input_options_with_data) {
             const details = `type of flag ${option} should be ${type_of_option} (and the received value is ${value})`;
             throw_cli_error(ManageCLIError.InvalidArgumentType, details);
         }
+        // special case for unset value (specified by '').
+        if (!LIST_UNSETTABLE_OPTIONS.includes(option) && value === '') {
+            const details = `flag ${option} is '' ${value} but it can't be unset.`;
+            throw_cli_error(ManageCLIError.UnsetArgumentIsInvalid, details);
+        }
```

Comment:
Still we are missing one check here, we are not checking leading space for unsettable options and faulty values assigned to unsettable flags due to the leading space.

Please check for below command(`new_buckets_path` will get assigned empty value here):

`$ sudo node src/cmd/manage_nsfs account add --name aayush2 --uid=9 --gid=9  --new_buckets_path= tmp/`

Reply:
I was not aiming to fix the leading spaces issue, please fix that on your PR